docs/sphinx/_toc.yml.in

```diff
@@ -174,8 +140,22 @@ subtrees:
           title: ROCm SMI LIB
         - url: https://rocm.docs.amd.com/projects/ROCmValidationSuite/en/latest/
           title: ROCm validation suite
```

Comment:
ROCm Validation Suite is not a library. Perhaps move it to Install?  

@LisaDelaney